transport/happyeyeballs_test.go

```diff
@@ -225,6 +225,7 @@ func TestHappyEyeballsStreamDialer_DialStream(t *testing.T) {
 		require.Equal(t, []string{"[2001:4860:4860::8888]:53", "8.8.8.8:53"}, dialedAddrs)
 	})
 
+	/* Commenting out flaky test until we have time to fix the race condition
```

Comment:
This is fine for now. But I think we can fix the problem by making the baseDialer wait a little bit on the first Dial.